collectd-openstack/README.md

```diff
@@ -0,0 +1,300 @@
+
+Metadata associated with the OpenStack plugin for collectd can be found [here](https://github.com/signalfx/integrations/tree/release/collectd-openstack). The relevant code for the plugin can be found [here](https://github.com/signalfx/collectd-openstack).
+
+- [Description](#description)
+- [Requirements and Dependencies](#requirements-and-dependencies)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+This is the SignalFx OpenStack plugin. Follow these instructions to install the OpenStack plugin for collectd.
+
+The [openstack-collectd](https://github.com/signalfx/collectd-openstack) plugin collects metrics from OpenStack components by hitting various endpoints. This plugin covers the following components:
+
+* Nova (Compute)
+* Cinder (BlockStorge)
+* Neutron (Network)
+
+Reference for OpenStack [Monitoring](https://wiki.openstack.org/wiki/Operations/Monitoring).
+
+#### FEATURES
+
+#### Built-in dashboards
+
+
+- **HYPERVISOR**: Provides a high-level overview of metrics for an OpenStack hypervisor.
+
+  [<img src='./img/openstack-hypervisor-dashboard-top.png' width=200px>](./img/openstack-hypervisor-dashboard-top.png)
+
+  [<img src='./img/openstack-hypervisor-dashboard-bottom.png' width=200px>](./img/openstack-hypervisor-dashboard-bottom.png)
+
+- **TENANT**: Provides metrics from an OpenStack project/tenant.
+
+  [<img src='./img/openstack-tenant-dashboard-top.png' width=200px>](./img/openstack-tenant-dashboard-top.png)
+
+  [<img src='./img/openstack-tenant-dashboard-bottom.png' width=200px>](./img/openstack-tenant-dashboard-bottom.png)
+
+- **NEUTRON**: Provides metrics from an OpenStack Neutron component.
+
+  [<img src='./img/openstack-neutron-dashboard.png' width=200px>](./img/openstack-neutron-dashboard.png)
+
+- **INSTANCE**: Provides metrics from an OpenStack compute instance.
+
+  [<img src='./img/openstack-instance-dashboard-top.png' width=200px>](./img/openstack-instance-dashboard-top.png)
+
+  [<img src='./img/openstack-instance-dashboard-bottom.png' width=200px>](./img/openstack-instance-dashboard-bottom.png)
+
+
+### REQUIREMENTS AND DEPENDENCIES
+
+#### Version information
+
+| Software  | Version        |
+|-----------|----------------|
+| collectd  |  4.9 or later  |
+| python | 2.6 or later |
+| Python plugin for collectd | (included with [SignalFx collectd agent](https://github.com/signalfx/integrations/tree/master/collectd)[](sfx_link:sfxcollectd)) |
+
+### INSTALLATION
+
+1. Download [collectd-openstack](https://github.com/signalfx/collectd-openstack). Place the `openstack_metrics.py`, `NovaMetrics.py`, `CinderMetrics.py`, and `NeutronMetrics.py` files into `/usr/share/collectd/collectd-openstack` directory.
+
+2. Modify the [sample configuration file](https://github.com/signalfx/integrations/tree/release/collectd-openstack/20-openstack.conf) for this plugin and copy to `/etc/collectd/managed_config` directory.
```

Comment:
Boy I must need more coffee or something! Each time I review this I see stuff I missed before. Steps 2 and 3 seem to be one step broken into 2 parts. Should it be "Modify the... for this plugin as described in.. below and then copy to...." ?